web/documentserver-example/python/src/utils/trackManager.py

```diff
@@ -159,12 +159,27 @@ def processForceSave(body, filename, usAddr):
         if forcesavePath == "":
             forcesavePath = docManager.getForcesavePath(filename, usAddr, True)
 
-    docManager.saveFile(download, forcesavePath)  # save document file
+    docManager.saveFile(data, forcesavePath)  # save document file
 
     if isSubmitForm:
         uid = body['actions'][0]['userid']  # get the user id
         historyManager.createMetaData(filename, uid, "Filling Form", usAddr)  # create meta data for forcesaved file
 
+        forms_data_url = body.get('formsdataurl')
+        if forms_data_url:
+            data_name = docManager.getCorrectName(fileUtils.getFileNameWithoutExt(filename) + ".txt", usAddr)
+            data_path = docManager.getStoragePath(data_name, usAddr)
+
+            forms_data = docManager.downloadFileFromUri(forms_data_url)
+
+            if data is None:
+                raise Exception("Document editing service didn't return forms_data")
```

Comment:
I think you wanted to use `forms_data` here.

Reply:
thanks